types/common.ts

```diff
@@ -35,8 +35,8 @@ export type ValueVariableType = {
   type: "discrete" | "continuous"
   name: string
   description: string
-  min: number
-  max: number
+  min: number | string
```

Comment:
It seems like a hack to allow min and max to be strings? Can it be avoided, e.g. by wrapping the object in a special validation type closer to the validation logic?

Reply:
Fixed by adding a ValueVariableInputType